### ISSUE TITLE
[a11y] Issue 27: Missing form label (Participation Page)

### ISSUE THREAD

**Issue by NewJerseyStyle** [NewJerseyStyle/polis#85]
**Description:**

A form control on the participation page is missing a corresponding label.

**Steps to Reproduce:**

1. Navigate to the participation page.
2. Inspect the form control using developer tools.

**Expected Behavior:**

Form controls should have associated labels.

**Actual Behavior:**

A form control lacks a label.

**Component/Area:** Participation page

**Source:** Auto evaluation

**Affected Elements:** label

**Comment by YumiChen** [comment]
The issue seems to be not present in home page
![Image](https://github.com/user-attachments/assets/0c657959-3cf4-4468-a4fa-09236f138191)
![Image](https://github.com/user-attachments/assets/66b78ece-d9c8-44c2-af73-95362a87fd9f)

Is it in sign-in page?
![Image](https://github.com/user-attachments/assets/66c6bf3b-9a04-40a1-853d-ffb3078a6fee)

**Comment by NewJerseyStyle** [comment]
The log says it is on Home page, maybe someone mixed `home page` and `sign-in page` 🤔 

Thanks for pointing it out 💖

**Comment by cplkake** [comment]
I believe the issue was originally meant for the closed participation page for the following reasons: 

- In the [log](https://docs.google.com/spreadsheets/d/176knLyqkJ0vxFMH-qtWi9J17-oJ3-ChdLQkISd59oOY/edit?gid=0#gid=0), the dev who identified this issue had helped identify 9 issues on the closed participation page right above this issue and perhaps they accidentally mislabelled it as a home page issue.
- There are no other issues identified for the sign-in page in the log which leads me to believe that noone has run an analysis and updated the log for the sign-in page even though there are many clear issues from your screenshots

Running WAVE on the closed participation page reveals that the page does have a missing form label but it is greyed-out meaning that the element is hidden by CSS.

**Comment by NewJerseyStyle** [comment]
@cplkake Do you think we should correct all `Home Page` issues to `Close Page`?

**Comment by cplkake** [comment]
The other `Home Page` issues are still `Home Page` issues so I think we should leave them be. Looking further into this issue on the `Closed Page`, since the form component is intentionally hidden visually, I will change this issue to a `Participation Page` issue. The same form component can be interacted with in a `Participation Page` and therefore this issue is pertinent to `Participation Page`.

**Comment by YumiChen** [comment]
@cplkake Thanks for clarifying the issue. Checked closed page in test environment, the issue should be fixed with our latest code
http://20.83.181.230/74fbvamy2y

![Image](https://github.com/user-attachments/assets/ebc7ac46-4af5-4825-a5a7-5a0ed82f9bda)
